docs/content/docs/guide/running.md

```diff
@@ -141,6 +141,11 @@ roses are red, violets are blue. pipeline are bound to flake by design.
 /test <pipelinerun-name>
 ```
 
+Note that the `/test <pipelinerun-name>` comment will start the `PipelineRun`
+regardless if the pipelinerun is matching the annotations. This let you have
+explicit control of PipelineRun for some type of Pull Requests controlled by the
+user instead of events.
+
```

Comment:
I tried to test manually 

couple of things

1. I had pipelinerun in `.tekton` dir which have 
```
metadata:
  name: another-pr-pac-demo
annotations:
    pipelinesascode.tekton.dev/on-event: "[pull_request, push]"
    pipelinesascode.tekton.dev/on-target-branch: "[main1]"
```
but gave wrong values so CI did not trigger then as per the doc added comment
```
/test another-pr-pac-demo
```
But CI did not start

Then commented those annotation and did `/test another-pr-pac-demo` then CI started

So can we explicitly add that details saying no annotations should be added :thinking: or let me know if my understanding is wrong here

2. I see `/retest` is not working if its not supported can we add that info as well ?

Reply:
I see yeah, thanks.. that would be a bug i need to fix i think  instead of just documentation..